src/index.js

```diff
@@ -168,18 +172,57 @@ function showGame() {
   updatePlayerScore();
 }
 
-
 function updatePlayerScore() {
-  player1Score.innerText = `${player1.score}`
-  player2Score.innerText = `${player2.score}`
-  player3Score.innerText = `${player3.score}`
+  player1Score.innerText = `${players[0].score}`
+  player2Score.innerText = `${players[1].score}`
+  player3Score.innerText = `${players[2].score}`
 }
 
 function displaySelectedClue(event) {
   let clickedCard = event.target.closest(".clue-card");
-  let selectedClue = clueInfo.find(clue => clue.id == clickedCard.id)
+  selectedClue = clueInfo.find(clue => clue.id == clickedCard.id)
   let selectedCategory = clueCategories.find(category => category.id === selectedClue.categoryId)
   $('.selected-clue-category').text(`${selectedCategory.category.split(/(?=[A-Z])/).join(" ").toUpperCase()}`);
   $('.selected-clue-points').text(`${selectedClue.pointValue}`);
   $('.question').text(`${selectedClue.question}`);
 }
+
+function evaluateGuess() {
+  let response;
+  $('.answer-response').css("display", "flex");
+  if ($(".player-guess").val().toUpperCase() === selectedClue.answer.toUpperCase()) {
```

Comment:
I like how you called on the toUpperCase method on the part of your function, it looks like you get past any barriers that would be put up when the player places any answers that are written weirdly with casing but this just makes all responses upper cased so I think that's awesome